context-free.md

```diff
@@ -2,21 +2,61 @@
 
 ##  Who is this programming language for?
```

Comment:
This tends to depend on the editor you're using, and how well it supports Markdown. If you're editing things on GitHub then what you've described should work, but isn't always the case.